src/Tika.Client/TikaClient.cs

```diff
@@ -0,0 +1,145 @@
+using System.Collections.Generic;
```

Comment:
I suggest we split this class into classes concerning the different concerns in tika